scripts/update_table_pairs.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+shopt -s globstar
+
+CWD="$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
+
+cd "$CWD/../"
+
+OUT=crates/db_schema/src/table_pairs.rs
+
+echo "// @generated by scripts/update_table_pairs.sh" > $OUT
+echo "use crate::schema::*;" >> $OUT
+echo "use diesel::allow_tables_to_appear_in_same_query as a;" >> $OUT
+
+for f in crates/**/**.rs src/**/**.rs
+do
+    python3 -c "import re; import itertools; print(''.join(set('a!({}, {});'.format(min(pair), max(pair)) for fn_chunk in open('$f').read().split('fn ') for pair in itertools.combinations(re.findall('[^A-Z]([a-z_]+)::table[^!]', fn_chunk) + re.findall('[^A-Z]([a-z_]+_actions)::', fn_chunk), 2) if pair[0] != 'pg_namespace' and pair[1] != 'pg_namespace' and pair[0] != pair[1])), end = '')" >> $OUT
```

Comment:
I would prefer if we dont have to depend on another language for development, only for this small script. Besides the code is not readable like this.